src/main/java/net/dv8tion/jda/api/entities/MessageReaction.java

```diff
@@ -402,9 +401,9 @@ public RestAction<Void> clearReactions()
 
     private String getReactionCode()
     {
-        return emote.isEmote()
-                ? emote.getName() + ":" + emote.getId()
-                : EncodingUtil.encodeUTF8(emote.getName());
+        return emoji.getType() == Emoji.Type.CUSTOM
+                ? emoji.getName() + ":" + ((CustomEmoji) emoji).getId()
+                : EncodingUtil.encodeUTF8(emoji.getName());
```

Comment:
`emoji.getAsReactionCode`

Reply:
as a note, we might not need this anymore because `.getEmoji().getAsReactionCode()`?